.mailmap

```diff
@@ -915,7 +915,9 @@ Raymond Wong <rayman_407@yahoo.com> <rayman@CoolGuy.(none)>
 Raymond Wong <rayman_407@yahoo.com> <rayman@CoolGuyJr.(none)>
 Raymond Wong <rayman_407@yahoo.com> <rayman@CoolGuyJr>
 Rehas Sachdeva <aquannie@gmail.com>
+Remco de Boer <29308176+redeboer@users.noreply.github.com>
 Remco de Boer <redeboer@gmx.com>
+Remco de Boer <redeboer@outlook.com>
```

Comment:
This records each name as a separate entry in the AUTHORs file. Which of these do you actually want to record as your name and email address for the AUTHORS file?

Reply:
29308176+redeboer@users.noreply.github.com

Reply:
Should I update that in a separate PR?

Reply:
I can fix it during the release. I just need to know which name/email it is that you want to use.

Reply:
Ok thanks. So just `Remco de Boer <29308176+redeboer@users.noreply.github.com>`